server/data/interfaces/whereaboutsApi/Appointment.ts

```diff
@@ -49,11 +49,12 @@ export interface PrePostAppointmentDetails {
   // PrePost Form values
   formValues?: {
     preAppointment?: string
-    preAppointmentLocation?: number
+    preAppointmentLocation?: string
     postAppointment?: string
-    postAppointmentLocation?: number
+    postAppointmentLocation?: string
     court?: string
     otherCourt?: string
+    hearingType?: string
```

Comment:
Question really - when BVLS switch is true, we should never let the user provide the `otherCourt` value - they must always select from the list. Do that hold up?